asterics-web-vue/src/libraries/util/help.js

```diff
@@ -0,0 +1,90 @@
+function loadTags() {
+  return fetch("https://api.github.com/repos/asterics/AsTeRICS/tags")
+    .then(res => res.json())
+    .then(out => {
+      let r = [];
+      if (!out.hasOwnProperty("message")) {
+        out.forEach(element => {
+          if (/^v\d(.\d){0,2}$/g.test(element.name)) {
+            r.push(element.name);
+          }
+        });
+      }
+      return r;
+    })
+    .catch(err => {
+      throw err;
+    });
+}
+
+async function asyncForEach(array, cb) {
+  for (let i = 0; i < array.length; i++) [await cb(array[i], i, array)];
+}
+
+const getContent = async l => {
+  return await getS(l);
+};
+
+const getS = async url => {
```

Comment:
what is `getS()`?

Reply:
not a good name, obviously. :grin:

It was short for "getStructure" or the like, but it will be or should be removed, as it is not necessary anymore. It was used to pull information of Folder from GitHub API, recursively.

Using git trees, however, recursion is not necessary anymore.